gnd/src/main/java/com/google/android/gnd/ui/common/FeatureHelper.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.ui.common;
+
+import android.content.Context;
+import androidx.annotation.Nullable;
+import com.google.android.gnd.R;
+import com.google.android.gnd.model.feature.Feature;
+import java8.util.Optional;
+
+public class FeatureHelper {
+
+  public static String getTitle(@Nullable Optional<Feature> featureOptional) {
```

Comment:
We generally don't add "Optional" to optionals unless there's a name clash - In a sense, in this case "optional" can be treated as a quality of the variable like type and nullability (e.g. we don't use "nameString" or "idNullable").

Reply:
Also, can we make this arg `@NonNull`?